resources/less/forum.less

```diff
@@ -1,27 +1,21 @@
 .ProfileConfigurePane {
     min-height: 100vh;
     padding: 20px 0;
-
     .Fields {
         margin: 10px 0;
-
         fieldset.Field {
+			max-width: 500px;
             margin: 0 5px;
             padding: 5px;
-
+			margin-bottom: 8px;
```

Comment:
Please indent with space. Use the editorconfig plugin to have this automatically taken care of :wink: 

Applies to the lines below as well